src/AddIns/Uno.UI.Lottie/ts/Uno.UI.Lottie.ts

```diff
@@ -85,6 +92,8 @@ namespace Uno.UI {
 				const frames = animation.getDuration(true);
 				const frame = frames * progress;
 				animation.goToAndStop(frame, true);
+				this.raiseState(animation);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use the parent class name instead of `this` when in a `static` context.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5436651)